src/main/cpp/t07_swap.cpp

```diff
@@ -19,5 +19,19 @@
 using namespace std;
 
 int t07_swap() {
-
+  int qua;
+  cin >> qua;
+  int *arr = new int[qua];
+  int swapper = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'swapper' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4739147)